### PR TITLE
Update images.md

### PR DESCRIPTION
**Pull request by yurj**
update scales

### CODE REVIEW

File: docs/classic-ui/images.md [hunk 1]
```diff
@@ -26,15 +26,19 @@ We will use the image object as context in the following examples.
...
-* large 768:768
-* preview 400:400
-* mini 200:200
+* huge 1600:65536
+* great 1200:65536
+* larger 1000:65536
+* large 800:65536
+* teaser 600:65536
+* preview 400:65536
+* mini 200:65536
```
Comment (stevepiercy) [inline docs/classic-ui/images.md:35]: 65536 is very high. Are you sure about these?

File: docs/classic-ui/images.md [hunk 2]
```diff
@@ -26,15 +26,19 @@ We will use the image object as context in the following examples.
...
 * thumb 128:128
 * tile 64:64
 * icon 32:32
 * listing 16:16
 
-You can add or change scales as you like.
+You can add or change scales as you like. The scales are defined in `plone.base.interfaces.controlpanel`.
```
Comment (stevepiercy) [inline docs/classic-ui/images.md]: One sentence per line.

```suggestion
You can add or change scales as you like.
The scales are defined in `plone.base.interfaces.controlpanel`.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
The values for height are much higher than I expected. Are these right?

**Comment by yurj** [comment]
One sentence per line.

File: docs/classic-ui/images.md (hunk 1 above)
Comment (yurj) [reply docs/classic-ui/images.md:35]: Yes, they came from plone.base.interfaces.controlpanel

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
It's OK to ignore the failing test and merge this PR. We hope that KitConcept resolves the expired certificate on https://plonedemo.kitconcept.com/ soon.

**Pull request merged**